benchmark_runner/common/template_operations/templates/winstress/internal_data/01_run_stress_template.ps1

```
@@ -0,0 +1,123 @@
+$ErrorActionPreference = 'Stop'
+
+$stressDir = 'C:\tools\stress'
+$pythonExe = 'C:\Program Files\Python312\python.exe'
+$stressScript = "$stressDir\stress.py"
+$reportFile = "$stressDir\stress_report.json"
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'reportFile' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)